### DISCUSSION TITLE
开源了支持ElectronBot的摄像头手势识别与多种SDK调用新玩法

### DISCUSSION THREAD

**Discussion by dbgba** [peng-zhihui/ElectronBot#213] [Show and tell]
我写了一个开源手势识别项目，调用了Mediapipe实现免部署的摄像头手势识别。使用的CPU支持，无需额外硬件。
同时又在@peng-zhihui 稚晖君SDK基础上更新了对AutoHotkey调用的支持，进而诞生了很多电脑自动化的交互玩法。

**摄像头手势识别项目**
https://github.com/dbgba/VisualGestureRecognition

**ElectronBotSDK改版玩法扩展与演示**
https://github.com/dbgba/ElectronBotSDK.AutoHotkey

以上两个项目主要是基于AutoHotkey开发，因为AHK是脚本语言，有着类似批处理.bat的工作方式。想要让ElectronBot做什么批量表情或动作，用记事本修改代码再运行即可实现。能更方便的快速玩转ElectronBot，又能对电脑自动化进行更多控制与深度操作。

**目前实现的玩法大致有：**
1.通过微信来控制ElectronBot

2.对智能家居的控制

3.量子纠缠Demo半成品

4.多个ElectronBot远程Tcp网络互动，手机网页控制ElectronBot

5.摄像头手势识别与ElectronBot动作同步

6.语音唤醒并调用ElectronBot表情等

7.连接ElectronBot时自动推送表情

8.热键快速调试ElectronBot

更多介绍或更新详见项目链接

**Comment by peng-zhihui** [comment]
感谢分享，可以考虑提交pr我帮你合入仓库

**Reply by dbgba** [reply]
好的，感谢大佬。我的SDK还有一些未完成部分，待我把示例和功能都完善后再提交pr
